packages/core/src/viewport/ViewportProvider.tsx

```diff
@@ -26,6 +26,7 @@ const ViewportProvider = ({ children }: ViewportProviderProps) => {
       );
 
       observer.observe(document.body);
+      setViewport(document.body.getBoundingClientRect().width);
```

Comment:
Will this break in SSR?

Reply:
I think it being inside `useIsomorphicLayoutEffect` means it shouldn't break